packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -813,6 +813,8 @@ function performConcurrentWorkOnRoot(root, didTimeout) {
         lanes = errorRetryLanes;
         exitStatus = renderRootSync(root, errorRetryLanes);
       }
+
+      executionContext &= ~RetryAfterError;
```

Comment:
This is the least intrusive change. I don't have all the context around what `executionContext` is used for. But I feel like it's missing a general "reset" step via `executionContext = NoContext`.

Reply:
Could you do `executionContext = prevExecutionContext` instead? It should be impossible for the `RetryAfterError` bit to already be set, since we prevent nested renders, but I wouldn't want this pattern to spread elsewhere.

Reply:
done: [`2c9bc48` (#21766)](https://github.com/facebook/react/pull/21766/commits/2c9bc487c0403186b343d6a7a09ba2766a82004a)